.github/workflows/database-upgrade.yml

```diff
@@ -0,0 +1,44 @@
+name: Database Upgrade Test
+
+on:
+  push:
+    branches:
+      - main
+      - '[0-9]+.[0-9]+'
+  pull_request:
+    branches:
+      - main
+      - '[0-9]+.[0-9]+'
+
+jobs:
+  upgrade_test:
+    runs-on: ubuntu-latest
+    container:
+      image: domjudge/gitlabci:24.04
+    services:
+      sqlserver:
+        image: mariadb
+        ports:
+          - 3306:3306
+        env:
+          MYSQL_ROOT_PASSWORD: root
+          MYSQL_USER: domjudge
+          MYSQL_PASSWORD: domjudge
+        options: --health-cmd="healthcheck.sh --connect --innodb_initialized" --health-interval=10s --health-timeout=5s --health-retries=3
+    steps:
+      - uses: actions/checkout@v4
+      - name: Wait and Import Database
+        run: |
+          apt-get update
+          apt-get install -y default-mysql-client
+          while ! mysqladmin ping -h"sqlserver" -u"root" -p"root" --silent; do
+            echo "Waiting for database connection..."
+            sleep 2
+          done
+          
+          mysql -hsqlserver -u"root" -p"root" < .github/jobs/data/dj733.sql
+      - name: Install DOMjudge
+        run: .github/jobs/baseinstall.sh default upgrade
+      - name: Check for Errors in the Upgrade
```

Comment:
Can you also run the download part from `webstandard.sh` and make sure that none of the pages result in a `HTTP500`? That's what most of the time happens.

Reply:
What test and role should I use to execute webstandard.sh? Or should I modify the script to add a new test parameter that only checks whether any page returns a 500 error?

Reply:
I would download as admin with a team role and I would just not run the `WCAG` or `W3C` tests but only do the downloading part with `wget`.

Reply:
If I only run webstandard.sh after the database upgrade is complete and use the admin role, will that be sufficient?

Reply:
Also fine by me.

Reply:
Okay, it's been added. Is this okay?